.github/workflows/deploy-dotorg.yml

```
@@ -0,0 +1,22 @@
+name: Deploy to WordPress.org
+
+on:
+  release:
+    types: [published]
```

I think we should listen to the `released` type instead, since `published` is not triggered when a pre-release is released. See [here](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-38): 

> * `published`: a release, pre-release, or draft of a release is published
> * `released`: a release or draft of a release is published, or a pre-release is changed to a release

```suggestion
    types:
      - released
```

---

Good catch @eugene-manuilov. Thanks.